### PR TITLE
layers-api: Adding optional CORS processing

### PR DESCRIPTION
**Pull request by akukhar**

### CODE REVIEW

**Comment by akukhar** [comment]
Please only pay attention to the 2nd commit only - "layers-api: Adding CORS processing". The first commit is a technical one.

File: helm/layers-api/templates/ingress.yaml [hunk 1]
```diff
@@ -14,7 +14,10 @@ metadata:
     nginx.ingress.kubernetes.io/server-snippet: |
 {{- else }}
     nginx.org/client-max-body-size: 5m
+    nginx.org/location-snippets: |
+      include CORS_02_location.inc.conf;
     nginx.org/server-snippets: |
+      set $cors_config_file '/etc/nginx/default.cors.json';
```
Comment (isemichastnov) [inline helm/layers-api/templates/ingress.yaml:20]: Are these files present in publicly available nginx ingress controller?
if not - this seems to break this ingress for anyone outside Kontur. I'd advice to parameterise this via a separate helm property.

FYI @Komzpa

Comment (akukhar) [reply helm/layers-api/templates/ingress.yaml:20]: https://konturio.slack.com/archives/C014AF8MFRD/p1671101846650499

Comment (akukhar) [reply helm/layers-api/templates/ingress.yaml:23]: I have added a compatibility layer with Flux

**Review by rkireyeu-kontur — APPROVED** [review]

**Pull request merged**